lib/perl/Genome/Site/TGI/Extension/Sys.pm

```diff
@@ -92,15 +88,14 @@ sub bsub_and_wait_for_completion {
             %this_cmd_bsub_args = ( cmd => $cmd );
         }
 
-        my $job_id = $class->bsub(post_exec_cmd => qq(echo $seq | netcat $hostname $port),
+        my $job_id = $class->bsub(
                                   %bsub_args,
                                   %this_cmd_bsub_args,
                                 );
-        $on_submit_cb->($seq, $job_id) if $on_submit_cb;
```

Comment:
Why are we removing this feature?  It's still used [in Genome::Model::Build::Unarchive](https://github.com/genome/genome/blob/f4f9c91170a5067476829fa65345cb45dd6d26b9/lib/perl/Genome/Model/Build/Command/Unarchive.pm#L195).